src/main/java/baritone/utils/schematic/format/defaults/LitematicaSchematic.java

```diff
@@ -0,0 +1,217 @@
+/*
+ * This file is part of Baritone.
+ *
+ * Baritone is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Baritone is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Baritone.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package baritone.utils.schematic.format.defaults;
+
+import baritone.utils.schematic.StaticSchematic;
+import net.minecraft.block.*;
+import net.minecraft.block.properties.IProperty;
+import net.minecraft.nbt.*;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.block.state.IBlockState;
+
+import org.apache.commons.lang3.Validate;
+import javax.annotation.Nullable;
+import java.util.*;
+
+/**
+ * @author Emerson
+ * @since 12/27/2020
+ */
+public final class LitematicaSchematic extends StaticSchematic {
+
+    public LitematicaSchematic(NBTTagCompound nbt) {
+        String regionName = (String) nbt.getCompoundTag("Regions").getKeySet().toArray()[0];
+        this.x = Math.abs(nbt.getCompoundTag("Regions").getCompoundTag(regionName).getCompoundTag("Size").getInteger("x"));
```

Comment:
The region size needs to be handled differently. The stored sizes per axis can be negative, if the "primary corner" (region origin" is not in the minimum corner of the region. (If they are negative, then they are at max -2, there should be no -1 values created by Litematica, as that is essentially the same as positive 1.)
![image](https://user-images.githubusercontent.com/2331050/154033845-acad8673-92c6-4dcd-bf13-b22e56d246a5.png)


I think for Baritone you can basically just use `Math.abs()` and then offset the origin if the values were negative for some axes. (No idea how you handle he positioning.) In Litematica this "freedom" in creating the selection with the primary corner in any relative corner causes quite some mess and complexity when handling the regions with rotations and mirroring etc.

Reply:
We always use the minimum corner for the position and don't have rotations, so just adding `size + 1` and then removing the sign from `size` should work fine.